cfg/1.11/node.yaml

```diff
@@ -1,23 +1,23 @@
 ---
 controls:
-version: 1.11
-id: 2
-text: "Worker Node Security Configuration"
+id: "2"
+version: "1.11"
```

Comment:
Doesn't matter much, but I think it reads better to have the version field ahead of the id